client/js/widget-manager.js

```diff
@@ -24,17 +24,21 @@ var Widgets = require('jupyter-js-widgets');
         // Register the comm target
         this.commManager.register_target(this.comm_target_name, this.handle_comm_open.bind(this));
 
-        // Validate the version requested by the backend.
-        var validate = (function validate() {
-            this.validateVersion().then(function(valid) {
-                if (!valid) {
-                    console.warn('Widget frontend version does not match the backend.');
-                }
-            }).catch(function(err) {
-                console.error('Could not cross validate the widget frontend and backend versions.', err);
-            });
-        }).bind(this);
-        validate();
+        // Validate the version requested by the backend -- required for ipywidgets
+        // If kernel name is not defined, default to python3
+        kernelname = kernelname.toLowerCase();
+        if (!kernelname || kernelname === 'python3' || kernelname === 'python2') {
```

Comment:
Discussed with @jhpedemonte . For now, we're not going to bother setting up decl widgets for scala since the example is only hello world. Adding whatever config is necessary for a scala decl widgets example to work can come as part of #203.